contracts-periphery/foundry.toml

```diff
@@ -2,4 +2,5 @@
 remappings = ['test/=test/']
 [ci]
 fuzz_runs = 10000
+remappings = ['test/=test/']
```

Comment:
As of a week or so ago foundry now adds this remapping by default, so we can remove these two remappings line. I'd also suggest adding `verbosity = 3` to the default profile. This will show traces for failing tests

Reply:
Removed unnecessary mappings and added verbosity here https://github.com/ScopeLift/umbra-protocol/commit/fc7758fd87a1ada3e4e3d85826f2914a317f7454#diff-1fab2f918de2e6b852e2d49b471dc962fe328740f1a48d85b4380befe1bac691=